modules/atomic-widgets/atomic-controls/types/select-control.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\AtomicControls\Types;
```

Comment:
do we really need to prefix everywhere?

```suggestion
namespace Elementor\Modules\AtomicWidgets\Controls\Types;
```

Reply:
Done.